calico/_includes/non-helm-manifests/csi-node-driver.yaml

```diff
@@ -0,0 +1,95 @@
+# This manifest installs the {{site.prodname}} CSI driver which
+# will be installed on every node in the cluster.
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: csi-node-driver
+  namespace: kube-system
```

Comment:
I can put it there, I put it in `kube-system` because if we were to follow a manifest install, `calico-system` wouldn't exist so I went with the one namespace that did exist. If I move it to `calico-system` I'll probably just add a duplicate version of the `calico-system` namespace to ensure that it's created whenever someone applies this manifest though.